### ISSUE TITLE
Error After Upgrading to 0.5.0 (You have to build Vue application)

### ISSUE THREAD

**Issue by ajdunn2** [codeceptjs/ui#229]
#### What are you trying to achieve?

- After updating this npm package from v0.4.7 to v0.5.0 the command ``npx codecept-ui --app`` no longer works

#### What do you get instead?

> Provide console output if related.

```bash
 ⚠️You have to build Vue application by `npm run build`
```

This appears to be due to the `dist` folder missing from the node package.

<img width="584" alt="047" src="https://user-images.githubusercontent.com/9711573/219255672-a9a0015e-06af-4a87-93ee-e6b594760d22.png">

<img width="601" alt="050" src="https://user-images.githubusercontent.com/9711573/219255680-61edc55b-4bfa-48a3-bbd5-f575215e9c58.png">

### Testing if dist folder is created locally
- Running the ``npm run build`` command on this repo causes the following error.
```
/  Building for production...
ERROR  Failed to compile with 1 error                                                                            
Syntax Error: TypeError: Cannot read properties of undefined (reading 'parseComponent')
```
- This appears to be due to the updated "vue-template-compiler v2.7.14" in this release not working with the version of vue in the package.lock file. Updating the version of vue 2.7.14 with `npm i vue@2.7.14` fixes this issues and allows it to build the project files into the dist folder. Moving this dist folder into this packages folder in node_modules fixes the above error message.

**Comment by guiyomh** [comment]
same for me

**Comment by stuta** [comment]
I tried to build this and got an error:
```
Building for production...Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:71:19)
    at Object.createHash (node:crypto:140:10)
    at module.exports (/Users/pasi/nc/nc-test/codecept-test/node_modules/.pnpm/@codeceptjs+ui@0.5.0_codeceptjs@3.4.1/node_modules/@codeceptjs/ui/node_modules/webpack/lib/util/createHash.js:135:53)
```

**Comment by bnpatel1990** [comment]
@DavertMik @Arhell  any updates on the above?? Thank you

**Comment by wclaytor** [comment]
I get the same error as above. I downgraded to '0.4.7' and it seems to be working (`"@codeceptjs/ui": "^0.4.7"`).

**Comment by a-stankevich** [comment]
This seems to be fixed in #231.

**Comment by avecAndo** [comment]
any news about the fix? Can we merge it soon?
